Wrapping/R/R.i

```diff
@@ -196,19 +196,27 @@ itk::simple::Image ArrayAsIm(SEXP arr,
 // Finer control will require putting swig code in the right
 // scope.
 
-%typemap("rtype") SEXP "function";
 %{
 #include "sitkRCommand.h"
 %}
 
+%ignore itk::simple::ProcessObject::AddCommand(itk::simple::EventEnum event, itk::simple::Command &cmd);
+
 %extend itk::simple::ProcessObject {
   int AddCommand( itk::simple::EventEnum e, SEXP obj )
  {
    // make sure that the CommandCallable is in fact callable
    if (!Rf_isFunction(obj))
      {
-           sitkExceptionMacro(<<"R object is not a function, "
-                              <<"or it has not been set.");
+       void *argp3 = 0 ;
+       int res3 = SWIG_R_ConvertPtr(obj, &argp3, SWIGTYPE_p_itk__simple__Command,  0 );
```

Comment:
Support for the direct sitk::Command class is not needed and does not need to be supported here.